airflow/utils/task_group.py

```diff
@@ -92,11 +92,15 @@ def __init__(
         ui_color: str = "CornflowerBlue",
         ui_fgcolor: str = "#000",
         add_suffix_on_collision: bool = False,
+        setup: bool = False,
+        teardown: bool = False,
     ):
         from airflow.models.dag import DagContext
 
         self.prefix_group_id = prefix_group_id
         self.default_args = copy.deepcopy(default_args or {})
+        self.setup = setup
+        self.teardown = teardown
```

Comment:
Should these be mutually exclusive, or can a task group be both setup and teardown?

Reply:
Should be mutually exclusive. Fixed